integration-tests/config/src/test/java/org/opennms/netmgt/config/WillItUnmarshalIT.java

```diff
@@ -220,7 +220,7 @@ private static void addFile(final Source source, final String file, final Class<
         addFile(Source.CONFIG, "email-northbounder-configuration.xml", EmailNorthbounderConfig.class, true, null);
         addFile(Source.CONFIG, "enlinkd-configuration.xml", EnlinkdConfiguration.class, false, null);
         addFile(Source.CONFIG, "eventconf.xml", Events.class, true, null);
-        addFile(Source.CONFIG, "eventd-configuration.xml", EventdConfiguration.class, true, null);
+        addFile(Source.CONFIG, "eventd-configuration.xml", EventdConfiguration.class, false, null);
```

Comment:
We shouldn't need to disable this check here. If the test is failing, we need to rework how the defaults are handled in the JAXB objects.